contract-project/waffle-config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "sourcesPath": "./contracts",
+  "targetPath": "./artifacts",
+  "npmPath": "./node_modules",
+  "solcVersion": "./node_modules/@eth-optimism/solc-transpiler",
+  "compilerOptions": {
+    "outputSelection": {
+      "*": {
+        "*": ["*"]
+      }
+    },
+    "executionManagerAddress": "0xA193E42526F1FEA8C99AF609dcEabf30C1c29fAA"
+  }
+}
```

Comment:
Adding waffle config because Buidler doesn't support `solcVersion` to be provided as a path to lib ☹️